cmd_states/run_test.py

```diff
@@ -0,0 +1,2 @@
+import Chandra.cmd_states
+Chandra.cmd_states.test()
```

Comment:
Also, regarding potential automated remote builds, we'd have a bunch more hurdles there.  Right now, these recipes are expecting to find git repositories in a local directory etc...